src/Symfony/Component/HttpKernel/DependencyInjection/ResettableServicePass.php

```diff
@@ -17,20 +17,18 @@
 use Symfony\Component\DependencyInjection\ContainerInterface;
 use Symfony\Component\DependencyInjection\Exception\RuntimeException;
 use Symfony\Component\DependencyInjection\Reference;
-use Symfony\Component\HttpKernel\EventListener\ServiceResetListener;
 
 /**
  * @author Alexander M. Turek <me@derrabus.de>
  */
 class ResettableServicePass implements CompilerPassInterface
 {
+    private $serviceName;
     private $tagName;
 
-    /**
-     * @param string $tagName
-     */
-    public function __construct($tagName = 'kernel.reset')
+    public function __construct($serviceName = 'services_resetter', $tagName = 'kernel.reset')
```

Comment:
Is there any point about making the service name configurable, whereas it's hardcoded in the Kernel?

Reply:
We made other passes configurable, so is this one. Doesn't hurt...

Reply:
`$serviceId`